rllib/evaluation/rollout_worker.py

```diff
@@ -544,11 +546,17 @@ def make_sub_env(vector_index):
             spaces=self.spaces,
             policy_config=policy_config)
 
-        # List of IDs of those policies, which should be trained.
-        # By default, these are all policies found in `self.policy_dict`.
-        self.policies_to_train: List[PolicyID] = policies_to_train or list(
-            self.policy_dict.keys())
-        self.set_policies_to_train(self.policies_to_train)
+        # Set of IDs of those policies, which should be trained. This property
+        # is optional and mainly used for backward compatibility.
+        self.policies_to_train: Optional[Set[PolicyID]] = None
+        self.is_policy_to_train: Callable[[PolicyID, SampleBatchType], bool]
```

Comment:
I feel like self._should_train_policy may be a better name than is_policy_to_train.
also, if these variables are private-ish, we should name them _policies_to_train and _should_train_policy
last thing is I feel like we may not want to keep a copy of policies_to_train on self. that just confuses things, since self.is_policy_to_train is the real source of truth.

Reply:
Yeah, this is just for backward compatibility. Some users may still use this property somewhere and those users are unlikely to ever change the `policies_to_train` list. So for them, it wouldn't matter, whether `self.is_policy_to_train` is the actual source of truth.

Reply:
But aren't functions that return bools always named: "is_...()"? Or "isPolicyToTrain()", etc..?

Reply:
I see. ok, will live with this then.
depending on the style guide :) I found our naming a bit less consistent sometimes, but that's a minor thing.